qiskit/__init__.py

```diff
@@ -44,7 +44,7 @@
 __path__ = pkgutil.extend_path(__path__, __name__)
 
 from .wrapper._wrapper import (compile, execute, load_qasm_string,
-                               load_qasm_file, least_busy, qobj_to_circuits)
+                               load_qasm_file, least_busy, qobj_to_circuits,)
```

Comment:
i think you have a copy paste error here

Reply:
Thanks for catching this